packages/library/src/commonMain/kotlin/io/realm/BaseRealm.kt

```diff
@@ -54,7 +54,7 @@ public abstract class BaseRealm internal constructor(
         }
 
     /**
-     * The current data version of this Realm and data fetched from it.
+     * The current version of this Realm and data fetched from it.
```

Comment:
I had `data` in there because we have quite a few version concepts: SDK itself, Schema and our "internal" version which this is. If data is the wrong word, I think we should rephrase this somehow.

Reply:
I was confused by the `data version` term in the first place. I tried to rephrase, if it doesn't compute in your end, I will just revert.